src/Microsoft.PowerShell.Commands.Utility/resources/UtilityCommonStrings.resx

```diff
@@ -174,4 +174,7 @@
   <data name="TypeNotSupported" xml:space="preserve">
     <value>'{0}' is not supported in this system.</value>
   </data>
+  <data name="NoZoneIdentifierFileStream" xml:space="preserve">
+    <value>"No 'Zone.Identifier' file stream. The file was already not blocked: {0}"</value>
```

Comment:
Should this be present tense e.g. "The file is already not blocked: ..."?  Maybe ask one of your documentation folks about the wording for this error message?  Sorry but error messages in PowerShell really need help and I'd rather we not contribute to the problem with poor error messages.  Do you have a "doc/grammar" person to review error messages?

Reply:
`The file is already unblocked: `

Reply:
@rkeithhill unfortunately at this time, our previous doc person left Msft.  Doc changes aren't breaking, so we can continue to improve them.  I agree reading it again that it should probably be present tense: The file is already not blocked.

@markekraus perhaps I'm a bit pedantic on this one, but `unblocked` implies the file was once blocked which may not have been the case here.

Reply:
@SteveL-MSFT You're right, and what Keith wrote is more accurate. but `already not blocked` sounds odd.. maybe `The file is not blocked`?

Reply:
@markekraus agree, that sounds better: The file is not blocked.